controllers/provisioners/eks/create.go

```diff
@@ -89,6 +89,14 @@ func (ctx *EksInstanceGroupContext) CreateScalingGroup(lcName string) error {
 	}
 	ctx.Log.Info("created scaling group", "instancegroup", instanceGroup.GetName(), "scalinggroup", asgName)
 
+	if spec.EKSConfiguration.SuspendedProcesses != nil {
```

Comment:
I think you can probably remove this if, and add a length check inside the method/s
```
if len(processesToSuspend) == 0 {
  return nil
}
```
inside `SetSuspendProcesses` - same for `SetResumeProcesses`